.github/workflows/pr_check_workflow.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         if: steps.job_successful.outputs.job_successful != 'true'
         uses: actions/setup-node@v2
         with:
-          node-version: "10.24.1"
+          node-version: "14.18.1"
```

Comment:
[14.18.2](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md#14.18.2) was just released on Nov 30th - do we want to bump to that now?

Reply:
I can do a quick test run with bumping to 14.18.2